source/docs/articles/local/drush-versions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Drush Versions on Pantheon
+description: Learn about Pantheon's default Drush version and how to implement site-local usage.
+keywords: Drupal drush, command line, drupal, terminus drush, cli, site-local drush
+---
+By default, Pantheon runs Drush 8.x on newly created Drupal sites and 5.x on sites created prior to [date]. We highly recommend running Drush 8.x locally when interacting with Drupal sites on Pantheon, but 7.x is also supported. Alias files generated by Pantheon are not compatible with 6.x, however you can execute commands locally with this version by including the `--strict=0` option.
```

Comment:
@greg-1-anderson any feedback on this first sentence? At the very least we plan on replacing "[date]" when Drush8 becomes default for new Drupal sites